### DISCUSSION TITLE
Actualisation branches

### DISCUSSION THREAD

**Discussion by Aycon3296** [community/community#47128] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hello, please tell me how to use Github branches.
I have a main branch and two auxiliary ones - main-dev and docs. In the first, I develop harmlessly for main, in the second, I write documentation for main. All three branches contain a LICENSE file. Suppose I want to change the license file in the main branch, I change it and make pull requests main -> main-dev, main -> docs. Github believes that the branches are out of sync and suggests that after that, make another pull-request back to main-dev -> main, docs -> main. What am I doing wrong? Why can't github see that the branches are already synced?

**Comment by jge162** [comment, marked as answer]
Hello @Aycon3296 So to understand your use case -> 

- You changed the LICENSE file in the main branch and made pull requests to merge the changes into main-dev and docs branches. 

- After the pull requests are merged, you can make pull requests to merge main-dev and docs branches back to the main branch.

- So based on that GitHub will suggest you do a pull request. That makes sense so all changes are merged and synced across all branches.

The pull request suggestion is legit.
